schema.go

```diff
@@ -1045,18 +1074,20 @@ func (s *Schema) genSingleIndexQuery(t *Table, g *Generator, r *rand.Rand, p Par
 		return nil
 	}
 
-	/* Once we have ALLOW FILTERING SUPPORT this can be applied
-	pkNum := p.Rand.Intn(len(t.PartitionKeys))
+	 // Once we have ALLOW FILTERING SUPPORT this can be applied
```

Comment:
this comment looks is not necessary anymore.

Reply:
agree. removed